apps/web/src/pages/quick-start/components/layout/BodyLayout.tsx

```diff
@@ -25,11 +24,10 @@ export function BodyLayout({ children }: { children: React.ReactNode }) {
 }
 
 const StyledBody = styled.div`
-  display: block;
-  flex: auto;
-  margin-bottom: ${FOOTER_HEIGHT}px;
+  height: calc(100vh - 180px);
   padding-top: 25px;
   padding-bottom: 25px;
+  overflow-y: auto;
```

Comment:
get started page body scrollable on smaller screens and footer sticks to the bottom